.github/workflows/js_test.yml

```diff
@@ -0,0 +1,20 @@
+name: Javascript testing
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  eslint:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Install
+      uses: CultureHQ/actions-yarn@master
+      with:
+        args: install
+    - name: Lint
```

Comment:
Not important, but think this should be called 'Test', not 'Lint' (linting is in another file, right)

Reply:
Whoops, you are correct.